examples/realtime_zshimming.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
what is the purpose of having this file in addition to `demo_realtime_zshimming.sh`? (they seem to overlap)